<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CStaffsDialog</class>
 <widget class="QDialog" name="CStaffsDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>458</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Staffs</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="minimumSize">
      <size>
       <width>496</width>
       <height>406</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="icon">
       <iconset resource="ObjectComposerXML.qrc">
        <normaloff>:/treble_clef.png</normaloff>:/treble_clef.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Staffs</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>171</width>
         <height>361</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="horizontalSpacing">
         <number>0</number>
        </property>
        <property name="verticalSpacing">
         <number>6</number>
        </property>
        <item row="1" column="0">
         <widget class="QToolButton" name="AddStaff">
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QToolButton" name="RemoveStaff">
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QListView" name="StaffList"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>0</y>
         <width>301</width>
         <height>361</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QWidget" name="formLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>281</width>
          <height>341</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::FieldsStayAtSizeHint</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Name:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="NameEdit"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Abbreviation:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="AbbreviationEdit"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Audible:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QCheckBox" name="AudibleCheckBox">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Voices:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <layout class="QGridLayout" name="gridLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <property name="horizontalSpacing">
            <number>0</number>
           </property>
           <property name="verticalSpacing">
            <number>6</number>
           </property>
           <item row="1" column="0">
            <widget class="QToolButton" name="AddVoice">
             <property name="minimumSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="text">
              <string>+</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="3">
            <widget class="QListView" name="VoiceList"/>
           </item>
           <item row="1" column="1">
            <widget class="QToolButton" name="RemoveVoice">
             <property name="minimumSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="icon">
       <iconset resource="ObjectComposerXML.qrc">
        <normaloff>:/measure.png</normaloff>:/measure.png</iconset>
      </attribute>
      <attribute name="title">
       <string>General</string>
      </attribute>
      <widget class="QWidget" name="formLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>471</width>
         <height>152</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::FieldsStayAtSizeHint</enum>
        </property>
        <property name="horizontalSpacing">
         <number>-1</number>
        </property>
        <property name="verticalSpacing">
         <number>8</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Hide Bar Numbers:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Bar Numbering Offset:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Master Staff:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QSlider" name="NoteSpaceSlider">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="minimum">
             <number>8</number>
            </property>
            <property name="maximum">
             <number>34</number>
            </property>
            <property name="pageStep">
             <number>4</number>
            </property>
            <property name="value">
             <number>16</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="SpacingLabel">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Note Spacing:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="HideBarNumbersCheckbox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="BarNumberOffsetSpinBox">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QComboBox" name="MasterStaffComboBox"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>471</width>
         <height>231</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QFontWidget" name="TempoBox">
          <property name="title">
           <string>Tempo Font</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QFontWidget" name="DynamicBox">
          <property name="title">
           <string>Dynamic Font</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFontWidget</class>
   <extends>QGroupBox</extends>
   <header>qfontwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>AddStaff</tabstop>
  <tabstop>RemoveStaff</tabstop>
  <tabstop>AddVoice</tabstop>
  <tabstop>RemoveVoice</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources>
  <include location="ObjectComposerXML.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CStaffsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CStaffsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AddStaff</sender>
   <signal>clicked()</signal>
   <receiver>CStaffsDialog</receiver>
   <slot>AddStaff()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RemoveStaff</sender>
   <signal>clicked()</signal>
   <receiver>CStaffsDialog</receiver>
   <slot>DeleteStaff()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>361</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AddVoice</sender>
   <signal>clicked()</signal>
   <receiver>CStaffsDialog</receiver>
   <slot>AddVoice()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>9</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RemoveVoice</sender>
   <signal>clicked()</signal>
   <receiver>CStaffsDialog</receiver>
   <slot>DeleteVoice()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>FillStaff(int)</slot>
  <slot>AddStaff()</slot>
  <slot>DeleteStaff()</slot>
  <slot>AddVoice()</slot>
  <slot>DeleteVoice()</slot>
  <slot>EditName()</slot>
  <slot>EditAbbreviation()</slot>
  <slot>EditHideBarNumbers()</slot>
  <slot>EditBarNumberOffset()</slot>
  <slot>EditNoteSpace()</slot>
  <slot>EditMasterStaff(int)</slot>
  <slot>EditAudible()</slot>
 </slots>
</ui>
